<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>updateForm</class>
 <widget class="QDialog" name="updateForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelStart">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDead">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Deadline</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>yyyy-MM-dd hh:mm</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plaitTextEditDead">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>yyyy-MM-dd hh:mm</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditFormat">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string>yyyy-MM-dd hh:mm or yyyy-MM-dd - format</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>cancel</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plaitTextEditTask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>task</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelStart_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDead_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>190</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Deadline</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEditStart_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>yyyy-MM-dd hh:mm</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plaitTextEditDead_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>yyyy-MM-dd hh:mm</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plaitTextEditTask_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>50</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="placeholderText">
    <string>task</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>C059</family>
     <pointsize>17</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Entry now</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
